Buy/Card.Client.swift

```diff
@@ -0,0 +1,112 @@
+//
+//  Card.Client.swift
+//  Buy
+//
+//  Created by Shopify.
+//  Copyright (c) 2017 Shopify Inc. All rights reserved.
+//
+//  Permission is hereby granted, free of charge, to any person obtaining a copy
+//  of this software and associated documentation files (the "Software"), to deal
+//  in the Software without restriction, including without limitation the rights
+//  to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+//  copies of the Software, and to permit persons to whom the Software is
+//  furnished to do so, subject to the following conditions:
+//
+//  The above copyright notice and this permission notice shall be included in
+//  all copies or substantial portions of the Software.
+//
+//  THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+//  IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+//  FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+//  AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+//  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+//  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+//  THE SOFTWARE.
+//
+
+import Foundation
+
+private struct Header {
+    static var accept      = "Accept"
+    static var contentType = "Content-Type"
+}
+
+public extension Card {
+    
+    public typealias VaultCompletion = (_ token: String?, _ error: Error?) -> Void
+    
+    /// The `Vault.Client` is a network layer designed to abstract the communication with the Shopify's
+    /// credit card server. For PCI-compliance reasons, this is a standalone service, separate from the
+    /// main `Graph.Client` service.
+    ///
+    public class Client {
+     
+        /// The `URLSession` backing all `Client` network operations. You may provide your own session when initializing a new `Client`.
+        public let session: URLSession
+        
+        internal let apiURL: URL = URL(string: "https://elb.deposit.shopifycs.com/sessions")!
```

Comment:
Is this always the same URL? With the old Checkout API, the vault URL would be part of the checkout object